source/ContextualPopup.js

```diff
@@ -542,6 +542,10 @@
 			this.inherited(arguments);
 			this.alterDirection();
 			this.showHideScrim(this.showing);
+
+			if(this.spotActivatorOnClose) {
+				enyo.Spotlight.spot(this.activator);
```

Comment:
@KunmyonChoi can you elaborate a little more on what we need to accomplish by checking focus inside the popup? I'm not understanding what the current short coming is with this solution.

Thanks!